.github/workflows/code.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"] # can add windows-latest, macos-latest
-        python: ["3.9", "3.10", "3.11"]
+        python: ["3.7", "3.9", "3.10", "3.11"]
```

Comment:
It seems very strange to not include 3.8 in this list.

Reply:
We test one version i.e. 3.8 with a non-editable install, and therefore there is no need to also test 3.8 with editable install.

Reply:
The non-editable install was supposed to be the lowest version, this should now be 3.7, with 3.8 tested as editable

Reply:
Changed.